website/docs/r/data_migration_service.html.markdown

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
can we add this to the sidebar too?

Reply:
Do you mean we should not add it into sidebar for now?

Reply:
It should be added to the website index erb so it shows up in the nav